packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3432,6 +3468,13 @@ export class ContainerRuntime
 			);
 		}
 	}
+
+	private get groupedBatchingEnabled(): boolean {
```

Comment:
Can you somehow add this to https://github.com/microsoft/FluidFramework/blob/b8ab5e42ebbfb5d820a11ed76bbf83739d583938/packages/runtime/container-runtime/src/containerRuntime.ts#L1446 for better observability? It would be very useful if only to rule out this particular feature if things go sideways.